packages/core/upload/admin/src/components/FilterList/tests/__snapshots__/index.test.js.snap

```diff
@@ -202,7 +202,7 @@ exports[`<FilterList /> renders and matches the snapshot 1`] = `
     <span
       class="c2 c3"
     >
-      type is not file
+      type is [object Object]
```

Comment:
I tested your branch and it works but this value is not good, please check why the snapshot gives you this value instead of is not file as before, thank you

Reply:
@simotae14 I updated the code by making the necessary changes. Please review at your convenience. Thank you!